doc/examples/applications/plot_morphology.py

```diff
@@ -191,7 +190,7 @@ def plot_comparison(original, filtered, filter_name):
 #*single-pixel wide skeleton*. It is important to note that this is
 #performed on binary images only.
 
-horse = io.imread(os.path.join(data_dir, "horse.png"), as_gray=True)
+horse = horse()
```

Comment:
Wouldn't it preferable to have variable names != functions?

Reply:
Most likely, but we should establish the bigger picture problems first.